website/docs/modules/gather_sample_evidence.md

```diff
@@ -6,20 +6,78 @@ slug: gse
 ---
 
 Runs raw evidence collection on each sample with the following SV callers: 
-Manta, Wham, and/or MELT. For guidance on pre-filtering prior to GatherSampleEvidence, 
+Manta, Wham, Scramble, and/or MELT. For guidance on pre-filtering prior to GatherSampleEvidence, 
 refer to the Sample Exclusion section.
 
-Note: a list of sample IDs must be provided. Refer to the sample ID 
-requirements for specifications of allowable sample IDs. 
+The downstream dependencies of the GatherSampleEvidence workflow 
+are illustrated in the following diagram.
+
+```mermaid
+
+stateDiagram
+  direction LR
+  
+  classDef inModules stroke-width:0px,fill:#00509d,color:#caf0f8
+  classDef thisModule font-weight:bold,stroke-width:0px,fill:#ff9900,color:white
+  classDef outModules stroke-width:0px,fill:#caf0f8,color:#00509d
+
+  gse: GatherSampleEvidence
+  eqc: EvidenceQC
+  gcnv: TrainGCNV
+  gbe: GatherBatchEvidence
+  gse --> eqc
+  gse --> gcnv
+  gse --> gbe
+  
+  class gse thisModule
+  class eqc, gcnv, gbe outModules
+```
+
+
+## Inputs
```

Comment:
How are you choosing which inputs to include? It seems like there are a lot included for GatherBatchEvidence, but only a small subset for GatherSampleEvidence. I am not necessarily opposed to just picking the most important inputs to describe, but the selection process is unclear to me

Reply:
We are aiming to extend documentation to cover all the required inputs and the most informative/commonly used outputs. We'll get there in multiple iterations and start with every input/output for which we have some resources. To document some of these arguments, we consulted with the source code and external resources, but that is not scalable. So, we certainly welcome folks with more knowledge on these inputs to chim in and extend docs as we're not familiar with every single input or output.